ruby/vampires.rb

```diff
@@ -0,0 +1,75 @@
+# method to determine the likelihood of being a vampire
+def determine_vampire (likes_garlic, needs_insurance, age_check, name)
+  result = "Results inconclusive."
+
+  if age_check && (likes_garlic || needs_insurance)
+    result = "Probably not a vampire."
+  end
+  if !age_check && (!likes_garlic || !needs_insurance)
+    result = "Probably a vampire."
+  end
+  if !age_check && !likes_garlic && !needs_insurance
+    result = "Almost certainly a vampire."
+  end
+  if name == "Drake Cula" || name == "Tu Fang"
+    result = "Definitely a vampire."
+  end
+  return result
+end
+
+puts "How many employees are being processed?"
+entries = gets.chomp.to_i
+
+iterations = 0
+
+until iterations == entries do
```

Comment:
Look into the times loop

http://ruby-doc.org/core-2.2.0/Integer.html#method-i-times